### PR TITLE
Added docs for GCS migration source

### PR DESCRIPTION
**Pull request by jonesphillip**
Added docs for GCS migration source

### CODE REVIEW

**Review by sdnts — APPROVED** [review]

File: content/r2/data-migration/super-slurper.md [hunk 1]
```diff
@@ -48,10 +43,13 @@ This setting determines what happens when an object being copied from the source
 We currently support copying data from the following cloud object storage providers to R2:
```
Comment (deadlypants1973) [inline content/r2/data-migration/super-slurper.md]:
```suggestion
Cloudflare currently supports copying data from the following cloud object storage providers to R2:
```
Would change 'we' to 'Cloudflare' here.

File: content/r2/data-migration/super-slurper.md [hunk 2]
```diff
@@ -48,10 +43,13 @@ This setting determines what happens when an object being copied from the source
...
 - Cloudflare R2
+- Google Cloud Storage (GCS) {{<inline-pill style="beta">}}
 
-## Create Amazon S3 credentials
+## Create credentials for storage providers
 
-To migrate objects from Amazon S3, Super Slurper requires access permissions to your bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions, Cloudflare recommends you create a user with a narrow set of permissions.
+### Amazon S3
+
+To copy objects from Amazon S3, Super Slurper requires access permissions to your bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions, Cloudflare recommends you create a user with a narrow set of permissions.
```
Comment (deadlypants1973) [inline content/r2/data-migration/super-slurper.md]: Is this bucket being referenced in first sentence an R2 bucket or S3 bucket? (Seems like the latter, but let's make this obvious)

Comment (deadlypants1973) [inline content/r2/data-migration/super-slurper.md]:
```suggestion
To copy objects from Amazon S3, Super Slurper requires access permissions to your S3 bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions, Cloudflare recommends you create a user with a narrow set of permissions.
```

File: content/r2/data-migration/super-slurper.md [hunk 3]
```diff
@@ -81,9 +79,35 @@ To create credentials with the correct permissions:
 
 You can now use both the Access Key ID and Secret Access Key when defining your source bucket.
 
+### Google Cloud Storage
+
+To copy objects from Google Cloud Storage (GCS), Super Slurper requires access permissions to your bucket. You can use the Google Cloud predefined `Storage Admin` role, but Cloudflare recommends creating a custom role with a narrower set of permissions.
```
Comment (deadlypants1973) [inline content/r2/data-migration/super-slurper.md]:
```suggestion
To copy objects from Google Cloud Storage (GCS), Super Slurper requires access permissions to your S3 bucket. You can use the Google Cloud predefined `Storage Admin` role, but Cloudflare recommends creating a custom role with a narrower set of permissions.
```

File: content/r2/data-migration/super-slurper.md [hunk 4]
```diff
@@ -81,9 +79,35 @@ To create credentials with the correct permissions:
...
+
+To create a custom role with the necessary permissions:
+1. Log in to your Google Cloud console.
+2. Go to **IAM & Admin > Roles**.
```
Comment (deadlypants1973) [inline content/r2/data-migration/super-slurper.md]:
```suggestion
2. Go to **IAM & Admin** > **Roles**.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**